src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -367,11 +367,22 @@ default int levelInterval() {
         return 1;
     }
 
+    @ConfigItem(
+        keyName = "levelMinValue",
+        name = "Minimum Skill Level",
+        description = "The minimum skill level required to warrant a notification. Useful for filtering out low-level notifications",
```

Comment:
```suggestion
        description = "The minimum skill level required to send a notification. Useful for filtering out low-level notifications",
```
warrant is correct, but it just doesn't feel like it fits in this situation, a boring basic word seems better here.